open-company/open-company-api
# RethinkDB pool

https://trello.com/c/L4PUZmVe

This adds DB connection pooling to our use of RethinkDB.

To test:

Follow the steps in the README for REPL usage. Notice the new step about starting the pool? Do all the steps still work? Good!

Start an API server on this branch. Access the web UI locally. Seem faster? Groovy time!

Run `lein test!`. All passing?

Review the pool.clj code and pool.clj tests.

Merge.
Carve a pumpkin.
Deploy to staging.
Hold onto your hat from all the speed.
